Client/Frontend/Browser/OpenInHelper.swift

```diff
@@ -91,11 +160,34 @@ class OpenPdfInHelper: NSObject, OpenInHelper, UIDocumentInteractionControllerDe
     private var docController: UIDocumentInteractionController? = nil
     private var openInURL: NSURL?
 
-    lazy var openInView: OpenInView = getOpenInView(self)()
+    lazy var openInView: OpenInView? = getOpenInView(self)()
+
+    lazy var documentDirectory: NSURL = {
+        return NSURL(string: NSTemporaryDirectory())!.URLByAppendingPathComponent(FileType.PDF.rawValue)
+    }()
+
+    private var filepath: NSURL?
```

Comment:
Nit: You can probably make this a `let` and non-optional since you have everything you need in the `init` call before calling super to setup this variable.